static/src/javascripts/projects/common/modules/commercial/commercial-audit.js

```diff
@@ -0,0 +1,31 @@
+define([
+    'common/utils/config',
+    'common/utils/report-error'
+], function (config, reportError) {
+
+    return {
+        init: init
+    };
+
+    function init() {
+        if (config.switches.commercialAudit) {
+            window.addEventListener('message', receiveMessage, false);
+        }
+
+    }
+
+    function receiveMessage(event) {
+        var origin = event.origin || event.originalEvent.origin;
+        var message = event.data;
+
+
+        if (message.indexOf('Tracker beacon: ') === 0) {
```

Comment:
the PR description isn't too helpful here, but it's causing a JS error, hope this isn't an essential feature
![image](https://cloud.githubusercontent.com/assets/7304387/15256183/22f9703a-1938-11e6-8c38-8d60f08fd9f6.png)